CodeCoverageQualityGate/CodeCoverageQualityService.ts

```diff
@@ -390,9 +390,13 @@ export class CodeCoverageQualityService {
 
 
     private getQualityGateIdentification() {
-        const referencePipeline: string = this.originalReferencePipelineName ? ", Reference pipeline: " + this.originalReferencePipelineName : "";
-        const referenceBuild: string = this.originalReferenceBuildNumber ? ", Reference build: " + this.originalReferenceBuildNumber : "";
-        return "Type: " + this.type + ", Threshold: " + this.threshold + referencePipeline + referenceBuild;
+        let text = "Type: " + this.type + ", Threshold: " + this.threshold;
+        if (this.type == TypeEnum.MODIFIED) {
```

Comment:
@yshi-parasoft This change make Quality Gate Identification more reasonable, since refenrence pipeline/build is needed only for MODIFIED type.